src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/webpack.config.vendor.js

```diff
@@ -2,6 +2,7 @@ var path = require('path');
 var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var extractCSS = new ExtractTextPlugin('vendor.css');
+const { AureliaPlugin, ModuleDependenciesPlugin } = require("aurelia-webpack-plugin");
```

Comment:
`ModuleDependenciesPlugin` is unused

Reply:
yeah, that's a leftover... should be removed. Much of the PR is not getting in, does it make sense to just create a new one?